app/src/main/java/edu/calpoly/android/recyclerviewdemo/AnimalUtils.java

```diff
@@ -20,6 +24,38 @@ public String getName() {
         public String getImageUrl() {
             return mImageUrl;
         }
+
+        public boolean isDangerous() {
+            return mDangerous;
+        }
+
+
+        public int describeContents() {
+            return 0;
```

Comment:
```suggestion
            return 1;
```

Reply:
```suggestion
            return 2;
```

Reply:
Actually, I don't think we should change this after all.